app/views/shared/internet/_mno_guidance.html.erb

```diff
@@ -9,9 +9,12 @@
   <li>cannot afford additional data</li>
 </ul>
 
+<p class="govuk-body">giffgaff, O2 and Tesco Mobile will not accept requests for students over the age of 16.</p>
```

Comment:
```suggestion
<p class="govuk-body">giffgaff and O2 will not accept requests for students over the age of 16.</p>
```

Reply:
good spot i will change to read from model instead